fixtures/fiber-fragments/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```

Comment:
Agree, we probably don’t want this except for a quick manual testing before committing this.